README.md

```diff
@@ -1,14 +1,13 @@
-# copy-component
-Creates a copy element from existing element using data-copy attribute. Places a copy icon next to the element. When clicking the icon, the value of the element is copied and a copied overlay is displayed over the element for a short duration.\
-![Without Component Image](/screenshots/Component_1.png?raw=true "Without Component")\
-![With Component Image](/screenshots/Component_2.png?raw=true "With Component")\
-![After Copy Image](/screenshots/Component_3.png?raw=true "After Copy")
+# Blockonomics Copy Component
+Creates a copy element from existing element using data-copy attribute. Places a copy icon next to the element. When clicking the icon, the value of the element is copied and a copied overlay is displayed over the element for a short duration.
 
 ## Usage instructions
-* Copy the component JS, CSS, and Icons to your project
+* Copy the component JS, CSS, and Icons to your project or use a CDN
 * Load the component CSS in the header of your html
-`<link rel="stylesheet" href="css/components.css">`
+`<link rel="stylesheet" href="css/bnomics-input.css">`
 * Load the component JS in the foorter of your html
-`<script src="js/copyToClipboard.js"></script>`
-* Add the data-copy attribute to your desired element
-`<input type="text" value="MyValue" data-copy>`
+`<script src="js/bnomics-input.js"></script>`
+* Add the element
+`<div class="bnomics-input" data-style="dark">Content</div>`
```

Comment:
We have changed from using the data-copy attribute to using the bnomics-input class unnecessarily. The only reason I can see for using classes over the data attribute is getElementsByClassName is better performance than querySelectorAll, however we are still using querySelectorAll to select by class.

Reply:
The class has only been added for consistent styling. The primary focus of this PR is to ensure a common approach that works with internal blockonomics elements.

data-copy isn't reliable because the component supports both the elements with/without data-copy attributes to copy the required content automatically.

Reply:
> The primary focus of this PR is to ensure a common approach that works with internal blockonomics elements.

The focus of this PR should be to add support for <span> and <textarea> onto the already existing support for <input>.